### ISSUE TITLE
[Discussion] `std.ZeroAddress`

### ISSUE THREAD

**Issue by leohhhn** [gnolang/gno#2316]
## Description

Should we have a `ZeroAddress` constant in Gno which will represent a universal burn address? This should be an address for which nobody can obtain the private key. I've found it missing multiple times when developing.

For example, in the case of GRC20 and other similar tokens, people usually send the tokens they want to burn to a zero address.

This "burn" functionality can be implemented in two ways as I see:
- Instead of having to provide a burn address to send tokens to, simply deduct the burn value from the caller's balance - simpler, but the responsibility of creating this kind of function is on the developer of the package/realm
- Have a universal `ZeroAddress` constant in Gno, for example in the `std` package, for which everyone knows the private key is not obtainable. 
 
WDYT?

**Comment by thinhnx-var** [comment]
@leohhhn :
In my view, we surely need `ZeroAddress`. This is the basis of tokens. I was suprised when I saw this issue :)
And I also think this implementation should be inspired by ETH `0x00 address`.

**Comment by moul** [comment]
Yes, the feature seems useful and will likely improve readability.

Just considering whether `std` is the most appropriate place. Perhaps we should explore ways to limit the size of the `std` API.

**Comment by leohhhn** [comment]
We could have a "consts" stdlib or similar for this if we had more constants. I think `std` is the right place right now, and as soon as #1695 is merged, we can work on the split of std and have a separate place for it.

**Comment by thinhnx-var** [comment]
@leohhhn 
yea I am working on this and will update our approach and implementation over here.

**Comment by grepsuzette** [comment]
Good idea.

Which is best:
- **ZeroAddress**
- **AddressZero**

`AddressZero` is more scary.
There's a movie called [Patient Zero](https://en.wikipedia.org/wiki/Patient_Zero_(film)).

**Comment by thinhnx-var** [comment]
@grepsuzette 
I am thinking about (maybe) the best approach and you are giving me one more thing to put in mind. Cool :D

**Comment by notJoon** [comment]
What's the status of the issue?

**Comment by leohhhn** [comment]
I believe the implementation that was tried was not good enough. I think this is still something we want.
